pyflakes/checker.py

```diff
@@ -870,6 +895,63 @@ def ignore(self, node):
     # additional node types
     COMPREHENSION = KEYWORD = FORMATTEDVALUE = handleChildren
 
+    def DICT(self, node):
+        # Complain if there are duplicate keys with different values
+        # If they have the same value it's not going to cause potentially
+        # unexpected behaviour so we'll not complain.
+        keys = [
+            self.convert_to_value(key) for key in node.keys
+        ]
+        values = [
```

Comment:
Okay, next push that will be changed to do it when it checks the values of each key later (only on ones that have the key).